### DISCUSSION TITLE
Docker container not responding

### DISCUSSION THREAD

**Discussion by jasonfen** [usememos/memos#4040] [Q&A]
Trying to get a test container up on a known-good docker instance, I can't curl localhost:port, but other containers are all working fine.

> docker-compose:
> `services:
>   memos:
>     image: neosmemo/memos:stable
>     container_name: memos
>     volumes:
>       - /home/jason/memos/:/var/opt/memos
>     ports:
>       - 5000:5320`

docker ps output:

CONTAINER ID   IMAGE                           COMMAND                  CREATED          STATUS                  PORTS                                                                                            NAMES
40cc59f98514   neosmemo/memos:stable           "./memos"                34 minutes ago   Up 34 minutes           5230/tcp, 0.0.0.0:5000->5320/tcp, [::]:5000->5320/tcp                                            memos

docker logs memos:
---
Server profile
version: 0.22.5
data: /var/opt/memos
dsn: /var/opt/memos/memos_prod.db
addr:
port: 5230
mode: prod
driver: sqlite
---
Version 0.22.5 has been started on port 5230
---
See more in:
👉Website: https://usememos.com
👉GitHub: https://github.com/usememos/memos
---

███╗   ███╗███████╗███╗   ███╗ ██████╗ ███████╗
████╗ ████║██╔════╝████╗ ████║██╔═══██╗██╔════╝
██╔████╔██║█████╗  ██╔████╔██║██║   ██║███████╗
██║╚██╔╝██║██╔══╝  ██║╚██╔╝██║██║   ██║╚════██║
██║ ╚═╝ ██║███████╗██║ ╚═╝ ██║╚██████╔╝███████║
╚═╝     ╚═╝╚══════╝╚═╝     ╚═╝ ╚═════╝ ╚══════╝

**Comment by RoccoSmit** [comment]
What is the response you get from your curl command? and can you navigate to the URL on your browser?
